isolates/pubspec.yaml

```diff
@@ -0,0 +1,10 @@
+name: isolates
+description: A sample command-line application.
+version: 1.0.0
+
+environment:
+  sdk: '>=2.14.1 <3.0.0'
```

Comment:
Is this a 2.15 feature? Should that be the lower bound?

Reply:
yes, good catch!